src/phantom/schema.py

```diff
@@ -22,7 +25,25 @@ class Schema(TypedDict, total=False):
     maxLength: Optional[int]
 
 
+def desc_from_docstring(cls) -> str:
+    # ds = (next(filter(lambda x: x.__doc__, cls.__mro__), cls).__doc__ or "").strip()
+    ds = (cls.__doc__ or "").strip()
```

Comment:
```suggestion
    docstring = (cls.__doc__ or "").strip()
```

issue: Same here, we can afford some verbosity.